app/Kconfig

```diff
@@ -188,6 +188,14 @@ endif
 #Split Support
 endmenu
 
+menu "Stenography Support"
+
+config ZMK_PLOVER_HID
```

Comment:
Once I merge in the NKRO PR in #726, it might be good to group those changes with this config option.

Reply:
Sounds good.